specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -507,6 +507,53 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/quotaType/{quotaType}/usages": {
+      "get": {
+        "summary": "Gets the list of subscription core usages for all skus used in a location, for a given type of quota.",
+        "description": "Gets the list of usages in cores for all skus used by a subscription in a given location, for a specific quota type.",
+        "operationId": "GetUsagesInLocation_list",
```

Comment:
I think they wouldn't want you to use both 'get' and 'list' in the operation name, but just choose one depending on whether its a 'get' or 'list' operation!